Library/Homebrew/sorbet/tapioca/compilers/tty.rb

```diff
@@ -0,0 +1,36 @@
+# typed: strict
+# frozen_string_literal: true
+
+require_relative "../../../global"
+require_relative "../../../utils/tty"
+
+module Tapioca
+  module Compilers
+    class Tty < Tapioca::Dsl::Compiler
+      # FIXME: Enable cop again when https://github.com/sorbet/sorbet/issues/3532 is fixed.
+      # rubocop:disable Style/MutableConstant
+      # This should be a module whose singleton class contains RuboCop::AST::NodePattern::Macros,
```

Comment:
I don't think this applies

Reply:
Oops, that was copypasta from the compiler I took inspiration from. 😬